01_dask.delayed.ipynb

```diff
@@ -70,12 +70,9 @@
    "cell_type": "markdown",
```

Comment:
<p>We use this "What just happened" phrasing elsewhere in the tutorial. Any reason to remove it?</p><p> </p><p>This may be a good point to re-introduce the "<code>Delayed</code> object, if we remove it from up above. Since this is the first time they're actually seeing a Delayed output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dask/dask-tutorial/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
"What just happened" doesn't really seem that common in the tutorial.  Here I was just trying to make the text more concise (there is some redundancy in the tutorial here since it is re-stating what .compute does, which was already described above, and ithe text is also re-stating that the task graph holds the data and functions, which was also already discussed above), separate topics (both .compute and .visualize are being discussed here in 'what happened' but we haven't looked at .visualize yet) and try to improve the logical flow of the text.  Showing the task graph with .visualize is a simpler illustration for the reader about what is happening under the hood.